### PR TITLE
Feature/upload user profile picture functionality

### PR DESCRIPTION
**Pull request by HarisShahidHameed**
Trello link : https://trello.com/c/3gmZHFmI/30-profile-picture-functionality

### CODE REVIEW

File: server/libs/userlibs.js [hunk 1]
```diff
@@ -252,4 +253,39 @@ const changePassword = function(data) {
     })
 }
 
-module.exports = {register, logIn, updateUser, logOut , changePassword, deleteUser, deActivateUser, getUser}
\ No newline at end of file
+const uploadPhoto = function(req) {
```
Comment (abdulhaqshah) [inline server/libs/userlibs.js:256]: using req??

File: server/libs/userlibs.js [hunk 2]
```diff
@@ -252,4 +253,39 @@ const changePassword = function(data) {
...
+    let id = req.params.uuid;
+    let form = new formidable.IncomingForm();
+    form.uploadDir = "public/images";
```
Comment (abdulhaqshah) [inline server/libs/userlibs.js]: where that image will be uploaded. are we saving the whole image in db or just a link

File: server/routes/user.js [hunk 3]
```diff
@@ -67,4 +67,12 @@ router.post('/user/changePassword', authenticate, (req,res) => {
     })
 })
 
+router.post('/user/uploadPhoto/:uuid', authenticate, (req,res) => {
+    userLibs.uploadPhoto(req).then((success) => {
```
Comment (abdulhaqshah) [inline server/routes/user.js:71]: req shouldn't be send to it

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: server/libs/userlibs.js (hunk 2 above)
Comment (abdulhaqshah) [reply server/libs/userlibs.js]: the link should b in constant or json file

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: server/libs/userlibs.js (hunk 2 above)
Comment (HarisShahidHameed) [reply server/libs/userlibs.js]: Done

**Pull request merged**